app/src/main/java/org/covidwatch/android/data/NTPTime.kt

```diff
@@ -0,0 +1,20 @@
+package org.covidwatch.android.data
+
+import com.lyft.kronos.AndroidClockFactory
+import com.lyft.kronos.KronosClock
+import org.covidwatch.android.di.AppContext
+import org.koin.core.KoinComponent
+import org.koin.core.inject
+
+class NTPTime {
```

Comment:
A separate wrapper is nice but having a static reference to a context is a bad thing that leads to memory leaks.

In `Koin` there is `androidApplication()` that returns `applicationContext`, have a look at any wrapper that use `androidApplication()` in `appModule`. For example `Notifications` -- we need something like that and then we will be injecting `NtpTime` anywhere we need it.

Reply:
Also, to make it clear -- I suggest to rename this class to `NtpTime` because of the [guidelines ](https://developer.android.com/kotlin/style-guide#camel_case)

Reply:
removed static reference to context, changed so it's passed into the constructor and never stored

renamed file

Reply:
Made NtpClock injectable. Injected into BaseCovidWatchApplication and started properly from onCreate()

Reply:
modified VerifyPositiveDiagnosisFragment to use NtpTime for date pickers